containerz/containerz.proto

```diff
@@ -299,3 +327,69 @@ message LogRequest {
 message LogResponse {
   string msg = 1;
 }
+
+message CreateVolumeRequest {
+  // name of the volume. If present, this system must create the volume with
+  // this name otherwise it must allocate a name.
+  string name = 1;
+
+  // The volume driver. If not specified, this defaults to the the standard
+  // linux mount operation (http://man7.org/linux/man-pages/man8/mount.8.html)
+  // and associated options.
+  string driver = 2;
+
+  // Options for the driver. The actual option keys and values are driver
```

Comment:
Can you give some examples here? Are you thinking about e.g., flags to `mount` that are supplied? I would prefer less "open" modelling here, so that you have something like:

```protobuf
enum Driver { 
  DS_UNSPECIFIED = 0;
  DS_MOUNT = 1;
  DS_NFS = 2;
  ...
}

message MountOptions {
  bool read_only = 1;
}

message NFSOptions {
  uint32 reconnect_time_seconds = 1;
}

Driver driver = 1;

oneof options {
  MountOptions mount = 2;
  NFSOptions nfs = 3;
  ...
}
```

where now we have much more explicitly specified options that might be implemented. I understand that this might need changes to `Containerz` when we're adding new drivers, but my concern is whether the very generic API we have here maps cleanly across different container runtimes. (Like, does this work for `hostPath` in k8s, or `configMap`?)

Reply:
For example

```
CreateVolumeRequest {
   name = "volume"
   driver = "local"
   options = {
      type = none
      o = bind
      device = /some/host/path
   }
}
```

Would result in a volume backed by the host path `/some/host/path` with the name `volume`.

I think this would work pretty well across across runtimes as all the details are passed to the runtime and the driver. If we are more explicit we would need to map that logic in the containerz server which could lead to large implementation.